problemsets-mat223/modules/module2-exercises.tex

```diff
@@ -53,7 +80,27 @@
 				parallel to $\mathcal P_2$, but which does not
 				intersect $\ell$? Why or why not?
 		\end{enumerate}
-
+        % Q4 Solutions
+        \begin{solution}
+            \begin{enumerate}
+                \item $\mathcal P_1\cap \ell = \Set*{\mat{1\\1\\0}}$
+                \item $\mathcal P_1\cap \mathcal P_2 = \Set*{x \in \R^3 : x=t\mat{1\\1\\3} + \mat{-1/2\\ 3/2\\ -1/2}, t \in \R}$
+                \item $\mathcal P_2\cap \ell = \Set*{\mat{2\\4\\1}}$
+                \item $\vec v = t\mat{0\\0\\1} + s\mat{1\\3\\1}, t,s \in \R$ \\ (There are many correct answers)
+                \item No. Since $P_2\cap \ell$ has one vector, the reduced row echelon form of $\mat{1 & 0 & 1 \\ 3 & 0 & 1 \\ 1 & 2 & 1 }$ has three pivot columns. \\ 
```

Comment:
Don't use `\\` to force a newline. Do a new paragraph if you really want things on a new line.

Reply:
I don't follow...can you try explaining again?